Assets/Scripts/WorldProperties.cs

```diff
@@ -91,6 +93,111 @@ void Start()
             clipShader = GameObject.FindWithTag("Ground").GetComponent<Renderer>().material.shader;
         }
 
+        /// <summary>
+        /// Cycle through the connected drones
+        /// </summary>
+        public static void SelectNextDrone()
+        {
+            Debug.Log("Selection next drone");
+
+            if(selectedDrone != null)
+            {
+                dronesQueue.Enqueue(selectedDrone);
```

Comment:
your design might want to be dronesQueue always has all the drones so don't enqueue here but enqueue every time you take a drone out in this fashion, that way you can preserve drone cycle order and allow for interrupts with selecting a drone via point and click.

Reply:
hmmm, we'll do that when we get there